app/src/Scheduler.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App;
+
+use App\Message;
+use Symfony\Component\Scheduler\Attribute\AsSchedule;
+use Symfony\Component\Scheduler\RecurringMessage;
+use Symfony\Component\Scheduler\Schedule;
+use Symfony\Component\Scheduler\ScheduleProviderInterface;
+
+#[AsSchedule]
+class Scheduler implements ScheduleProviderInterface
+{
+    public function getSchedule(): Schedule
+    {
+        $schedule = new Schedule();
+
+        $schedule->add(RecurringMessage::every('12h', new Message\SynchronizeConnectors()));
```

Comment:
J’ai testé mais "12h" n’est pas un intervalle valide :

```suggestion
        $schedule->add(RecurringMessage::every('12 hours', new Message\SynchronizeConnectors()));
```